netstandard/ref/System.Security.Cryptography.X509Certificates.cs

```diff
@@ -47,6 +72,16 @@ public enum StoreName
         TrustedPeople = 7,
         TrustedPublisher = 8,
     }
+    public sealed partial class SubjectAlternativeNameBuilder
+    {
+        public SubjectAlternativeNameBuilder() { }
+        public void AddDnsName(string dnsName) { }
+        public void AddEmailAddress(string emailAddress) { }
+        public void AddIpAddress(System.Net.IPAddress ipAddress) { }
+        public void AddUri(System.Uri uri) { }
+        public void AddUserPrincipalName(string upn) { }
```

Comment:
can we give it a better name?

Reply:
I would assume that we can't - if it's already being used in the main framework.